src/store/setup-key/actions.ts

```diff
@@ -28,13 +28,16 @@ const actions = {
       'DELETE_SETUP_KEY_FAILURE'
   )<RequestPayload<string>, DeleteResponse<string | null>, DeleteResponse<string | null>>(),
   setDeleteSetupKey: createAction('SET_DELETE_SETUP_KEY')<DeleteResponse<string | null>>(),
+  resetDeletedSetupKey: createAction('RESET_DELETE_SETUP_KEY')<null>(),
```

Comment:
Codacy has a fix for the issue: [Replace `'RESET_DELETE_SETUP_KEY'` with `"RESET_DELETE_SETUP_KEY"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  resetDeletedSetupKey: createAction("RESET_DELETE_SETUP_KEY")<null>(),
```